packages/react/src/table/TableRow.tsx

```diff
@@ -1,67 +1,70 @@
 import { PropsWithChildren } from 'react';
 import { boxPalette, tokens } from '../core';
+import { theme } from '../ag-branding';
 import { useTableContext } from './TableContext';
 
 export type TableRowProps = PropsWithChildren<{
-	/** Indicates the current selected state of the table row. */
-	selected?: boolean;
 	/** The row index of the table row. */
 	'aria-rowindex'?: number;
+	/** Style the row when one cell contains an error. */
+	hasRowError?: boolean;
```

Comment:
Opted not to go with an extendable prop, e.g. `tone` since additional states like "info", "warning" etc I couldn't think of any appropriate use-cases for. We also have the `selected` prop which performs similar stylistic changes and is a boolean style prop. Worst case, later on we could map `hasRowError` to `tone="error"` as an alias.